docs/reference/aggregations/bucket/frequent-items-aggregation.asciidoc

```diff
@@ -0,0 +1,159 @@
+[[search-aggregations-bucket-frequent-items-aggregation]]
+=== Frequent items aggregation
+++++
+<titleabbrev>Frequent items</titleabbrev>
+++++
+
+A bucket aggregation which finds frequent item sets. Finding frequent item sets 
+is a form of association rules mining that identifies items that are often occur 
+together. Items that are frequently purchased together or log events that tend 
+to co-occur are examples of frequent item sets. Finding frequent item sets helps 
+to discover relationships between different data points (items) in a data set 
+and to create association rules based on those relationships.
+
+
+==== Syntax
+
+A `frequent_items` aggregation looks like this in isolation:
+
+[source,js]
+--------------------------------------------------
+"frequent_items": {
+  "minimum_set_size": 3,
+  "fields": [
+    {"field": "my_field_1"},
+    {"field": "my_field_2"}
+  ]
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+.`frequent_items` Parameters
+|===
+|Parameter Name |Description |Required |Default Value
+|`field` |(array) Fields to analyze. | Required |
+|`minimum_set_size` | (integer) The minimum size of one item set. | Optional | `1`
+|`minimum_support` | (integer) The minimum support of one item set. | Optional | `0.1`
+|`size` | (integer) The number of top item sets to return. | Optional |
+|===
+
+
+[discrete]
+[[frequent-items-minimum-support]]
+==== Minimum support
+
+Support represents the frequency of an item set normalized between 0 
+and 1. It is calculated by dividing the number of documents containing the item 
+set by the total number of documents.
+
+For example, if a given item set is contained by five documents and the total 
+number of documents is 20, then the support of the item set is 5/20 = 0.25. 
+Therefore, this set is returned only if the minimum support is 0.25 or lower. As 
```

Comment:
> Therefore, this set is returned only if the minimum support is 0.25 or lower. As 

I don't think I understand this section. It sounds like you'd only set this property if you already know the ratio of documents that contain your "frequent items" (how would you know that?). And by setting the property to this ratio, you're helping it quickly ignore/prune more of the items from its results. Is that the gist?

Reply:
You set the property to return only the item sets with the minimum support you are interested in. You don't know which item sets will be returned, but you know that – for example – every item sets with a minimum support that less than 0.5 is not interesting for you, so you set minimum support of 0.5 to get the items you are interested in.